Packs/CommonTypes/Layouts/layoutscontainer-Windows_Registry_KeyV2.json

```diff
@@ -0,0 +1,782 @@
+{
+    "fromVersion": "6.10.0",
+    "edit": {
+        "sections": [
```

Comment:
Layout is missing the grid from the "new form"
![image](https://user-images.githubusercontent.com/61933087/222441530-f694c4b6-1e96-45c1-adf1-42dee0b58c5f.png)

Reply:
Updated